test/ex_doc/formatter/html/autolink_test.exs

```diff
@@ -10,6 +10,9 @@ defmodule ExDoc.Formatter.HTML.AutolinkTest do
     test "autolinks fun/arity in docs" do
       assert project_doc("`example/2`", %{locals: ["example/2"]}) == "[`example/2`](#example/2)"
 
+      assert project_doc("`example/1,2`", %{locals: ["example/1", "example/2"]}) ==
+               "[`example/1,2`](#example/1)"
```

Comment:
could you add a test when we reference `example/1,2` but only `example/1` function exists? I think the behaviour should be (untested):

```elixir
assert project_doc("`example/1,2`", %{locals: ["example/1"]}) ==
         "`example/1,2`"  
```

What do you think?

In some cases we also trigger warnings if the function we try to link to doesn't exist (not for local functions though) so it would be good to test that with multiple arities too.

Reply:
Definitely! I did put this on my to-do list but somehow overlooked it :disappointed:   
Added a test for missing one of two/three function definitions and a test for the warning emitted when the module exists but [one of the] function[s] does not.